controllers/grid/files/query/ManageQueryNoteFilesGridHandler.inc.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @file controllers/grid/files/query/ManageQueryNoteFilesGridHandler.inc.php
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ManageQueryNoteFilesGridHandler
+ * @ingroup controllers_grid_files_query
+ *
+ * @brief Handle the query file selection grid
+ */
+
+import('lib.pkp.controllers.grid.files.SelectableSubmissionFileListCategoryGridHandler');
+
+class ManageQueryNoteFilesGridHandler extends SelectableSubmissionFileListCategoryGridHandler {
+	/**
+	 * Constructor
+	 */
+	function ManageQueryNoteFilesGridHandler() {
+		import('lib.pkp.controllers.grid.files.query.QueryNoteFilesCategoryGridDataProvider');
+		parent::SelectableSubmissionFileListCategoryGridHandler(
+			new QueryNoteFilesCategoryGridDataProvider(),
+			WORKFLOW_STAGE_ID_EDITING,
+			FILE_GRID_DELETE|FILE_GRID_VIEW_NOTES|FILE_GRID_EDIT
+		);
+
+		$this->addRoleAssignment(
+			array(
+				ROLE_ID_SUB_EDITOR,
+				ROLE_ID_MANAGER,
+				ROLE_ID_ASSISTANT
```

Comment:
Authors will not have the power to start queries right? I don't remember the spec really well...